cloud/migrate-to-cloud/hypershift.md

```diff
@@ -0,0 +1,172 @@
+maximum amount of storage available in the target database before you start
+migration.
+
+<Experimental />
+
+In preliminary testing, Hypershift migrated 60&nbsp;GB of data in 9 to 12 minutes,
+and 1&nbsp;TB of data in under 4 hours. You can continue reading from your
+source database during this time, though performance could be slower. If you
+write to tables in your source database during the migration, the new writes
+might not be transferred to Timescale Cloud. To avoid this problem, fork your
+database and migrate your data from the fork. To avoid this problem, see the
+section on [migrating an active database]
+(<http://docs.timescale.com/cloud/latest/migrate-to-cloud/#migrate-an-active-database>).
+
+<highlight type="important">
+If you have a very large database, and Hypershift is going to have to run for a
+very long time to migrate it, for example, a day or more, ensure that you have a
+very stable network connection. Hypershift is not able to recover if the network
+connection is interrupted.
+</highlight>
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+*   Signed up for your [free Timescale Cloud account][cloud-install].
+*   Installed [Docker][docker-install].
+
+<highlight type="cloud" header="Run all tutorials free" button="Try for free">
+Your Timescale Cloud trial is completely free for you to use for the first
+thirty days. This gives you enough time to complete all the tutorials and run
+a few test projects of your own.
+</highlight>
+
+## Download the Hypershift container
+
+<Tabs label="Download Hypershift">
+
+<Tab title="MacOS">
+
+<procedure>
+
+### Downloading the Hypershift container on MacOS
+
+1.  Open the Docker app on your local machine to start the service.
+
+1.  At the command prompt, pull the latest Hypershift container from Dockerhub:
+
+    ```bash
+    docker pull timescale/hypershift
+    ```
+
+</procedure>
+
+</Tab>
+
+<Tab title="Linux">
+
+<procedure>
+
+### Downloading the Hypershift container on Linux
+
+1.  Start the Docker service:
+
+    ```bash
+    sudo service docker start
+    ```
+
+1.  At the command prompt, pull the latest Hypershift container from Dockerhub:
+
+    ```bash
+    docker pull timescale/hypershift
+    ```
+
+</procedure>
+
+</Tab>
+
+</Tabs>
+
+## Run Hypershift
+
+In preparation for running Hypershift, you need to determine which tables need
+to be converted to hypertables, and which tables need to be compressed during
+the migration. Hypertables must have a unique column labelled `time`.
+
+Hypershift uses a YAML configuration file to determine how to set up your new
+Timescale database. You can create your own file, or use the example file as a
+starting point. You need these details to complete your configuration file:
+
+*   The schema you want to use
+*   The name of the database you are migrating
+*   The name of the `time` column in your database
+*   The chunk time interval to use
```

Comment:
It might be helpful to link to our docs for how to determine the chunk time interval? Or do we assume that the user knows this at this point?

Reply:
We could provide a link to the appropriate How To.